### ISSUE TITLE
EnDeclaration

### ISSUE THREAD

**Issue by vito-c** [ensime/ensime-vim#126]
I am getting some inconsistency when jumping to declarations. I am not sure if ensime-server supports all use cases because from what I could tell the byte positions were correct.

````scala  
override def someFunction(thing: SomeDependencyType): Future[Unit] = {
    import foo.bar.implicit.SomeImplicit
````
Whenever I `EnDeclaration` it takes me to the SomeImplicit declaration.

**Comment by vito-c** [comment]
I am also unable to jump to declaration `TheseThings`
````scala
    val things = Seq(
      TheseThings(
        thingOne = "thingone",
        thingTwo = "thingtwo"
      )
  )
````

**Comment by vito-c** [comment]
@yazgoo I was also wondering if there is a way to send a request to the ensime server on the commandline so that I could try and figure out why information for the parameter type isn't coming back correctly.

**Comment by vito-c** [comment]
could this be related to the ensime-server looking at old code? how do you force a refresh?

**Comment by yazgoo** [comment]
You force a refresh by running sbt gen-ensime and relaunching ensime server (i.e. right now relaunching vim)

**Comment by jvican** [comment]
I think this is fixed in #147. Please, try it and close in case it is.
